test/client/unit/pages/AssessmentConfirmation.spec.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import { mount, shallow } from 'enzyme';
+import { mount } from 'enzyme';
```

Comment:
it now has to be mount because of `DocumentTitle`? Does that slow the tests down a lot?